logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/scenario/OnIncomingCall.kt

```diff
@@ -36,7 +37,7 @@ class OnIncomingCall(
             callRepository.createCall(
                 conversationId = conversationId.toConversationId(),
                 status = CallStatus.INCOMING,
-                callerId = userId,
+                callerId = federatedIdMapper.parseToFederatedId(userId),
```

Comment:
Here we are parsing the id, we need to do the same I think in the other `callRepository.createCall()`  when we start a call ?